src/game/camera.h

```diff
@@ -90,6 +90,13 @@
 #define CAM_MODE_LAKITU_WAS_ZOOMED_OUT  0x02
 #define CAM_MODE_MARIO_SELECTED         0x04
 
+/**
+ * Object fields which are global to former camera.c objects
+ **/
+
+/* End Birds */
+#define /*0x104*/ oEndBirdCutsceneVars9PointX OBJECT_FIELD_F32(0x1F)
```

Comment:
Why have it here?

Reply:
It was a remnant of the behaviors that were still in camera.c, but now I guess it can just go to global_object_fields